### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by davegomez**
This PR address issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
 # AWeber FlexBox Grid
+AWeber FlexBox Grid is born from AWeber's need to have a responsive and modern grid system to use on its website and web application. This is why we used FlexBox grid as the base for this project but using SCSS as a preprocessor language to provide maintainability and ease of development.
```
Comment (jordangehman) [inline README.md]: `AWeber FlexBox Grid is a responsive modern grid system for creating structured layouts in both websites and web applications. Based on flexbox properties, it is a customized version of the infamous [Flexbox Grid](http://flexboxgrid.com/) and uses SCSS as a preprocessor language to facilitate development and maintenance.`

`If you find the grid useful in your projects, please give us a star and/or contribute :)`

Not sure if that last sentence is necessary.

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+
+## Development
+* Fork and clone the **AWeber FlexBox Grid** repository
+* Install project’s dependencies
+* Create a branch for your changes using [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
+* Make your changes and document them properly using [SASSDoc](http://sassdoc.com/) guidelines
+* Lint your changes and create a PR for your changes pointing to `develop` branch
+
+### Install dependencies
+We use [Yarn](https://yarnpkg.com/) as a package manager, please install it before start working on the project.
```
Comment (jordangehman) [inline README.md]: `Install [Yarn](https://yarnpkg.com/) package manager before working on the project.`

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+
+### Bower
+TBD…
+
+### Production CSS | CDN
+TBD…
+
+## Development
+* Fork and clone the **AWeber FlexBox Grid** repository
+* Install project’s dependencies
```
Comment (jordangehman) [inline README.md]: `Install the project's dependencies.`

File: README.md [hunk 4]
```diff
@@ -1 +1,43 @@
...
+TBD…
+
+### Bower
+TBD…
+
+### Production CSS | CDN
+TBD…
+
+## Development
+* Fork and clone the **AWeber FlexBox Grid** repository
```
Comment (jordangehman) [inline README.md]: Add a period to the end.

File: README.md [hunk 5]
```diff
@@ -1 +1,43 @@
...
+* Lint your changes and create a PR for your changes pointing to `develop` branch
+
+### Install dependencies
+We use [Yarn](https://yarnpkg.com/) as a package manager, please install it before start working on the project.
+```bash
+yarn
+```
+
+### Build documentation
+We use SASSDoc to build the project’s documentation although this is not a required step prior to publish a PR.
```
Comment (jordangehman) [inline README.md]: `Use [SASSDoc](http://sassdoc.com/) to build the project’s documentation. _Not required to open a PR._`

File: README.md [hunk 6]
```diff
@@ -1 +1,43 @@
...
+```bash
+yarn docs
+```
+
+### Lint code changes
+Is very important to lint your code changes before creating the PR so the continuous integration service allows to merge the PR.
```
Comment (jordangehman) [inline README.md]: `Lint your changes before opening a PR. If you don't, the Continuous Integration (CI) service won't allow your PR to be merged.`

File: README.md [hunk 7]
```diff
@@ -1 +1,43 @@
...
+```bash
+yarn lint
+```
+
+### Build distribution files
+If you need or want to check the production output files you can use this command:
```
Comment (jordangehman) [inline README.md]: `Check the production output files, if needed.`

File: README.md [hunk 8]
```diff
@@ -1 +1,43 @@
...
+
+### Production CSS | CDN
+TBD…
+
+## Development
+* Fork and clone the **AWeber FlexBox Grid** repository
+* Install project’s dependencies
+* Create a branch for your changes using [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
+* Make your changes and document them properly using [SASSDoc](http://sassdoc.com/) guidelines
+* Lint your changes and create a PR for your changes pointing to `develop` branch
```
Comment (jordangehman) [inline README.md]: `* Lint your changes before opening a PR.`
`* Open a PR comparing your branch with the `develop` branch as the base.`

File: README.md [hunk 9]
```diff
@@ -1 +1,43 @@
...
+### Bower
+TBD…
+
+### Production CSS | CDN
+TBD…
+
+## Development
+* Fork and clone the **AWeber FlexBox Grid** repository
+* Install project’s dependencies
+* Create a branch for your changes using [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
```
Comment (jordangehman) [inline README.md]: Add a period to the end.

File: README.md [hunk 10]
```diff
@@ -1 +1,43 @@
...
+* Make your changes and document them properly using [SASSDoc](http://sassdoc.com/) guidelines
```
Comment (jordangehman) [inline README.md]: `* Make your changes and document them using the [SASSDoc](http://sassdoc.com/) guidelines.`

**Review by jordangehman — CHANGES_REQUESTED** [review]
Hopefully these changes are clear. I rewrote a lot of it to be more straightforward and consistent.

**Comment by davegomez** [comment]
@jordangehman proposed changes addressed!

File: README.md [hunk 11]
```diff
@@ -12,32 +12,33 @@ TBD…
...
-* Install project’s dependencies
-* Create a branch for your changes using [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
-* Make your changes and document them properly using [SASSDoc](http://sassdoc.com/) guidelines
-* Lint your changes and create a PR for your changes pointing to `develop` branch
+* Fork and clone the **AWeber FlexBox Grid** repository.
+* Install the project's dependencies.
+* Create a branch for your changes using [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/).
+* Make your changes and document them using the [SASSDoc](http://sassdoc.com/) guidelines.
+* Lint your changes before opening a PR.
+* Open a PR comparing your branch with thedevelopbranch as the base.
```
Comment (jordangehman) [inline README.md]: The word "develop" should be formatted as code. Looks like a copy/paste issue.

**Review by jordangehman — CHANGES_REQUESTED** [review]
Just one error.

File: README.md (hunk 11 above)
Comment (davegomez) [reply README.md]: Are you accusing me of copy/pasta? 🤔

Fixed!

**Review by jordangehman — APPROVED** [review]

**Pull request merged**